libs/cart/src/models/cart-item.ts

```diff
@@ -20,7 +20,13 @@ export interface DaffCartItem {
   sku: string;
   name: string;
   qty: number;
+  /**
+   * The price per item, not including cart discounts.
+   */
   price: number;
+  /**
+   * The total for the entire row, not including cart discounts.
```

Comment:
I think we should mention that this includes category/product discounts

Reply:
changed

Reply:
*apply

Reply:
changed